languages/csharp/exercises/concept/randomness/RandomnessTests.cs

```diff
@@ -0,0 +1,30 @@
+using Xunit;
+using System;
+
+public class RandomnessTests
+{
+    [Fact]
+    public void RollDie()
+    {
+        var player = new Player();
+        bool outOfRange = false;
+        for (int i = 0; i < 100; i++)
+        {
+            var pips = player.RollDie();
+            if (pips <= 0 || pips > 18)
+            {
+                outOfRange = true;
+                break;
+            }
+        }
+        Assert.False(outOfRange);
```

Comment:
Was there any reason for not doing:

```suggestion
        var player = new Player();
        for (var i = 0; i < 100; i++)
        {
            Assert.InRange(player.RollDie(), 1, 18);
        }
```